tools/windows/DatadogAgentInstaller/CustomActions/Native/NativeMethods.cs

```diff
@@ -467,7 +468,7 @@ public void AddToGroup(SecurityIdentifier securityIdentifier, SecurityIdentifier
             var name = new StringBuilder();
             uint cchReferencedDomainName = 0;
             var referencedDomainName = new StringBuilder();
-            ReturnCodes err = ReturnCodes.ERROR_NONE_MAPPED;
+            var err = ReturnCodes.ERROR_NONE_MAPPED;
```

Comment:
That is a bit harsh IMHO

Reply:
I feel that we should remove it but I do not think it is big deal. The only problem I see that it ALWAYS forcing you to do it even when you do not want to do it. Perhaps it can be excepted with nolint comment but it is bit odd too. What about your thinking?